homeassistant/components/netgear_lte/icons.json

```diff
@@ -0,0 +1,39 @@
+{
+  "entity": {
+    "sensor": {
+      "sms": {
+        "default": "mdi:message-processing"
+      },
+      "sms_total": {
+        "default": "mdi:message-processing"
+      },
+      "upstream": {
+        "default": "mdi:ip-network"
+      },
+      "connection_text": {
+        "default": "mdi:radio-tower"
+      },
+      "connection_type": {
+        "default": "mdi:ip"
+      },
+      "service_type": {
+        "default": "mdi:radio-tower"
+      },
+      "register_network_display": {
+        "default": "mdi:radio-tower"
```

Comment:
Is the change from `mdi:web` to `mdi:radio-tower` intentional for `register_network_display`?

Reply:
It wasn't